interface-elements-for-web/articles/rich-text-editor/tables.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Tables
+
+---
+# Tables
+You can create and edit tables in documents. 
+
+Click the **Table** button in **Insert** tab to insert a table. Then select the table size and click the corresponding square in the drop-down box.
+
+![RichTextEditor-Tables-InsertTable](../../images/rich-text-editor-tables-insert-table.png)
+
+## Resize Table Element
+Drag cell borders to change column widths or row heights. 
+
+You can use [ruler](viewing-and-navigating/show-horizontal-ruler.md) markers to resize columns. 
+
+![RichTextEditor-Tables-Resizing](../../images/rich-text-editor-tables-resizing.png)
+
+## Context Tabs
+Use buttons in the **Design** and **Layout** context tabs to customize a table's design and layout settings. These tabs appear when a pointer is in a table element.
+
+![RichTextEditor-Tables-DesignTab](../../images/rich-text-editor-tables-design-tab.png)
+
+![RichTextEditor-Tables-LayoutTab](../../images/rich-text-editor-tables-layout-tab.png)
+
+
+## Context Menu
+Right-click a table element to display a context menu. The menu commands allows you to modify table properties and layout.
```

Comment:
modify table properties and layout -> modify the table's properties and layout.

Reply:
'commands allows'?

Reply:
Changed to 'You can use the menu's commands to modify the table's properties and layout.'